bin/main.rb

```diff
@@ -1,60 +1,79 @@
 #!/usr/bin/env ruby
+# rubocop:disable Metrics/BlockLength
 
-# rubocop:disable Lint/UselessAssignment
+require_relative '../lib/game_logic'
+require_relative '../lib/player'
+
+# Initialize the game and two players instances
+game = TicTacToe.new
+player1 = Player.new
+player2 = Player.new
 
 # 1. Prompt both users to give their names
 puts "Player 1, what's your name?"
-player1 = gets.chomp
+player1.name = gets.chomp
 puts '=============================='
 puts "Awesome! Player 2, what's your name?"
-player2 = gets.chomp
+player2.name = gets.chomp
 puts '=============================='
-puts "Welcome #{player1} and #{player2}"
+puts "Welcome #{player1.name} and #{player2.name}"
+
+# Ask Users if they want to see game instructions.
 puts 'Do you want to see game instructions? [y/n]'
 response = gets.chomp
-# 2. If yes, print game instructions! If not, proceed with the following step.
 puts 'Instructions' if response == 'y'
```

Comment:
- [ ] This should print the instructions, right? Since right now is not doing anything, you could remove it, or actually print the instructions to play the game (which would be cool).

![image](https://user-images.githubusercontent.com/61479648/94554070-e4c89080-022f-11eb-9bf6-ab6a477aab0d.png)